moviepy/tools.py

```diff
@@ -65,39 +65,41 @@ def is_string(obj):
     except NameError:
         return isinstance(obj, str)
 
+
 def cvsecs(time):
-    """ Will convert any time into seconds.
-    Here are the accepted formats:
-
-    >>> cvsecs(15.4) -> 15.4 # seconds
-    >>> cvsecs( (1,21.5) ) -> 81.5 # (min,sec)
-    >>> cvsecs( (1,1,2) ) -> 3662 # (hr, min, sec)
-    >>> cvsecs('01:01:33.5') -> 3693.5  #(hr,min,sec)
-    >>> cvsecs('01:01:33.045') -> 3693.045
-    >>> cvsecs('01:01:33,5') #coma works too
+    """ Will convert any time into seconds. 
+    
+    If the type of `time` is not valid, 
+    it's returned as is. 
+
+    Here are the accepted formats::
+
+    >>> cvsecs(15.4)   # seconds 
+    15.4 
+    >>> cvsecs((1, 21.5))   # (min,sec) 
+    81.5 
+    >>> cvsecs((1, 1, 2))   # (hr, min, sec)  
+    3662  
+    >>> cvsecs('01:01:33.045') 
+    3693.045
+    >>> cvsecs('01:01:33,5')    # coma works too
+    3693.5
+    >>> cvsecs('1:33,5')    # only minutes and secs
```

Comment:
this is a new format supported. Is it ok, right?

Reply:
Looks fine to me